backend/testfiles/execution/language.tests

```diff
@@ -238,10 +238,18 @@ module Pipes =
   |> (+) 3)) = 10
 
   // check typing
+
+  // TODO: resolve this test 
+  // (let x = 1 in
+  // (5
+  // |> x
+  // |> x
+  // |> (+) 3)) = Test.typeError_v0 "Expected a function value, got something else: 1"
```

Comment:
What's the issue with this test?

Reply:
The issue is that `x` is not a callable (it is literal `5`), or it is OK?

Reply:
Right - you shouldn't be able to `5 |> x` here - you can't "call" `1` with `5` as a parameter.
So the text looks correct to me.